sources/packages/backend/libs/utilities/src/queue.constant.ts

```diff
@@ -4,6 +4,7 @@
 export enum QueueNames {
   StartApplicationAssessment = "start-application-assessment",
   CancelApplicationAssessment = "cancel-application-assessment",
+  VirusScanProcessor = "virus-scan",
```

Comment:
can we change it to ClamAVScanProcessor or something like FileVirusScanProcessor? - because ClamAV means Clam Anti Virus